contracts/libraries/wrapppers/BiMapWrapper.sol

```diff
@@ -0,0 +1,29 @@
+pragma solidity ^0.5.0;
+
+import "../BiMap.sol";
+
+
+contract BiMapWrapper {
```

Comment:
A slightly more elegant way for testing might have been to write tests in solidity instead of js. Then the wrapper would not be needed here.

Anyways, your way is also fine.

Reply:
Yes this would make a lot of sense. I can try it sometime.

Reply:
@josojo unfortunately, after some investigation, I see that even open zeppelin are testing their libraries the way I have done here.